anomalib/utils/callbacks/tiler.py

```diff
@@ -0,0 +1,83 @@
+"""Tiler Callback."""
+
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+
+from typing import Optional, Sequence, Union
+
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import Callback
+
+from anomalib.models.components import AnomalyModule
+from anomalib.pre_processing.tiler import Tiler
+
+__all__ = ["TilerCallback"]
+
+
+class TilerCallback(Callback):
```

Comment:
Should we change this to `TilerConfigurationCallback`, or would it look weird from the CLI? The current name suggests that this callback applies the tiling operations.

Reply:
yes, I guess we could consider this approach after geting the first cli approach ready